packages/trpc/server/routers/viewer/slots.tsx

```diff
@@ -1,4 +1,5 @@
 import { SchedulingType } from "@prisma/client";
+import { performance } from "perf_hooks";
```

Comment:
```suggestion
import { performance } from "@calcom/lib/server/perfObserver";
```

Nice catch, although we might want to import the pre-configured one.